sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -48,12 +48,7 @@
     public AlterTableAnalyzedStatement analyze(AlterTable node, Row parameters, String defaultSchema) {
         Table table = node.table();
         TableInfo tableInfo = schemas.getTableInfo(TableIdent.of(table, defaultSchema));
-
-        // If the returned TableInfo is not of type DocTableInfo, then it is a system table.
-        if  (!(tableInfo instanceof DocTableInfo)) {
-            Operation.blockedRaiseException(tableInfo, Operation.ALTER);
-        }
-
+        Operation.blockedRaiseException(tableInfo, Operation.ALTER_BLOCKS);
```

Comment:
think we get get rid of that line by passing in the `Operation.ALTER_BLOCKS` into `getTableInfo`